### ISSUE TITLE
Failed validation

### ISSUE THREAD

**Issue by 974115250** [Dar9586/NClientV2#526]
Your issue may already be reported!
Please search on the [issue tracker](../) before creating one.

## Expected Behavior
After verification, load the picture normally

## Current Behavior
Force to jump to the login interface and display invalid response after login

## Steps to Reproduce (for bugs)

1.
2.
3.
4.

## Your Environment

* Version used:
* Operating System version:

* NHentai link (if relevant):

**Comment by DatBoiOrly** [comment]
Yeah I'm having the exact same issue also it's sad since it has been going strong for a long time :(
